.storybook/test-storybook.config.js

```diff
@@ -8,65 +8,65 @@ const configuration = {
         },
         'Core components/AreaChart': {
             'data labels config': {
-                'readySelector': '.screenshot-wrapper div:nth-child(8) .highcharts-container'
+                'readySelector': '.s-ready-wrapper-done'
```

Comment:
as we changed the component name it would be good to adjust selector name as well `ScreenshotReadyWrapper`